UnityAds/NSError+UnityAdAdapter.h

```diff
@@ -0,0 +1,25 @@
+//
+//  UnityAdAdapterError.h
+//  MoPub-TestApp-Local
+//
+//  Created by Vita Solomina on 2021-08-04.
+//  Copyright © 2021 Unity Ads. All rights reserved.
+//
```

Comment:
Forgot to mention, we don't commit this type of comments

Reply:
good catch, fixed it!